mill-universal-packager/src/io/github/hoangmaihuy/mill/packager/universal/UniversalPackagerModule.scala

```diff
@@ -33,4 +34,19 @@ trait UniversalPackagerModule extends PackagerModule {
     PathRef(zip)
   }
 
+  // Create a local directory with all the files laid out as they would be in the final distribution
+  def stage: T[PathRef] = T {
```

Comment:
I think it's better to name it `universalStage` to avoid naming conflict with other plugins

Reply:
this is just from the `sbt-native-packager` naming style.
but indeed `universalStage` is more clear.